public/data/communities.json

```
@@ -274,5 +274,30 @@
         "linkedin": "jimmy-adaro"
       }
     ]
+  },
+  {
+    "id": "7",
+    "name": "HackTandil",
+    "shortDescription": "Un espacio comunitario para conocer personas que hacen tecnología en Tandil.",
+    "fullDescription": "Un espacio comunitario para conocer personas que hacen tecnología en Tandil.",
+    "location": {
+      "lat": -371918.01,
+      "lng": -590759.38
```

This location is incorrect. Is this fixed location the correct one?

```suggestion
      "lat": -37.191801,
      "lng": -59.075938
```

---

Ups, was merged 😛 

Pls, open another PR fixing this 😉